pkg/dids/trustbloc_did_creator.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package dids
```

Comment:
This should be `did` instead of `dids`(plural). Same is followed in aries framework as well.

Reply:
It would clash with that package, forcing us to provide aliases.

Reply:
It shouldn't be an issue as we do use aliases in many places.

Reply:
can we change it

Reply:
done